pipelines/TechAgnostic/Utility/LRConvertBCF.wdl

```diff
@@ -1,15 +1,20 @@
 version 1.0
 
-############################################################################################
-## A workflow that converts a BCF file into a .vcf.gz file. Meant to temporarily handle some
-## transient issues stemming from the LRJointCallGVCFs workflow. Should be removed eventually.
-############################################################################################
-
 import "../../../tasks/VariantCalling/GLNexus.wdl" as GLNexus
 import "../../../tasks/Utility/Hail.wdl" as Hail
 import "../../../tasks/Utility/Finalize.wdl" as FF
 
 workflow LRConvertBCF {
+
+    meta {
```

Comment:
we need to create a ticket for this, given the comment there.

Reply:
#395